app/app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const {getTopics} = require('./controller.js')
+const app = express()
+app.use(express.json())
+app.use((error, request, response, next) =>
+{
+    if(error)
+    {
+        response.status(500).send({error})
+    }
+})
+
+app.get('/api/topics', getTopics);
+
+app.listen(9550, () => {console.log('listening on port 9550')})
```

Comment:
No need for `app.listen( )` at the moment as supertest sets up its own listener for its tests.